bindings/gumjs/gumdukinterceptor.c

```diff
@@ -183,6 +189,10 @@ _gum_duk_interceptor_init (GumDukInterceptor * self,
   // []
   _gumjs_duk_add_properties_to_class (ctx, "InvocationContext",
       gumjs_invocation_context_values);
+  self->cached_invocation_context_in_use = FALSE;
```

Comment:
Let's move the _in_use initialization last, so they follow the order in the struct.